components/program_manager/config/routes.rb

```diff
@@ -1,5 +1,14 @@
 ProgramManager::Engine.routes.draw do
   scope :api do
-    resources :transactions
+    get :vancouver_time_now, to: 'vancouver_time#now'
```

Comment:
I'd consider rephrasing to 
```suggestion
    get :current_vancouver_time, to: 'vancouver_time#now'
```